### ISSUE TITLE
SIMBAD not currently working

### ISSUE THREAD

**Issue by guillochon** [astropy/astroquery#891]
Hi all, it looks like SIMBAD queries are not currently working, but it might be on SIMBAD's end (I have e-mailed CDS support). The default mirror is `http://simbad.u-strasbg.fr/simbad/sim-script` and that page shows the current error message when you load the page:

`
!! A problem occured with the script file. It may be too large (max: 31457280 bytes) null/null/null
`

The error thrown by astroquery is:

`
/Users/james/miniconda3/lib/python3.6/site-packages/astroquery/simbad/core.py:136: UserWarning: Warning: The script line number 3 raised an error (recorded in the `errors` attribute of the result table): Identifier not found in the database : NAME sample
`

May be nothing that can be done at the moment, but I figured I'd open an issue to highlight the problem.

**Comment by keflavich** [comment]
Could you provide an example query that causes this error?

**Comment by bsipocz** [comment]
@guillochon - Could you please post a failing example? I've tried the first two from the docs page and it worked as expected.

**Comment by guillochon** [comment]
Sure here's my query:

```
customSimbad = Simbad()
customSimbad.ROW_LIMIT = -1
customSimbad.TIMEOUT = 120
customSimbad.add_votable_fields('otype', 'sptype', 'sp_bibcode', 'id')
table = customSimbad.query_criteria('maintype=SN | maintype="SN?"')
```

**Comment by bsipocz** [comment]
OK, I can confirm I see the same error.

**Comment by keflavich** [comment]
I think this is on our end:
```
>>> customSimbad._last_query.data
{'script': '\nvotable {main_id,coordinates,otype,sptype,sp_bibcode,id}\nvotable open\nquery sample    \nvotable close'}
```
It looks like the criteria didn't make it into the query at all.

**Comment by guillochon** [comment]
I'll also mention that I very recently upgraded my stack, it was functional on an older version of the stack (which had been running daily for the past year), so this might be a newly-introduced issue.

**Comment by keflavich** [comment]
This is a problem with `*args` and `**kwargs`:
```
def query_criteria_async(self, cache=True, *args, **kwargs):
```
Your parameters are getting passed into the `cache` kwarg, which is wrong.

**Comment by bsipocz** [comment]
Indeed, it got messed up with adding ``cache`` in 1e762c57d0acc9581df3f09b4e549c9db31b0692

**Comment by guillochon** [comment]
I can confirm that downgrading to astroquery 0.3.4 makes it work again.

**Comment by pllim** [comment]
The obvious workaround is to add a `cache=True` to your current query.

**Comment by guillochon** [comment]
@pllim That workaround actually does not work, I get an empty table returned (I cleared the cache first too). Downgrading to 0.3.4 seems to be the only way to actually yield a table.

**Comment by pllim** [comment]
:thinking: Hmm. Interesting. Thanks for trying anyway!

**Comment by keflavich** [comment]
I have a fix here:
https://github.com/keflavich/astroquery/tree/simbad_issue891
but I want to attach it to this issue and am getting this error using hub:
```
Error creating pull request: Unprocessable Entity (HTTP 422)
fork_collab Fork collab can't be granted by someone without permission
```
so I want to understand that before making a redundant PR

**Comment by bsipocz** [comment]
@guillochon - due to the nature of the unpacking of args and kwargs, you need to pass the ``cache=True`` as an arg (kargs are passed directly to the payload and thus won't work). The following works for me:

```
 table = customSimbad.query_criteria(True, 'maintype=SN | maintype="SN?"')
```

**Comment by bsipocz** [comment]
@keflavich - In the few cases I turned an issue to a PR, I used ``issue2pr.py`` from astropy-tools and it worked.

**Comment by bsipocz** [comment]
However when you attach commits to someone else PR, that messes a bit up the GH searches. E.g. this will come up as @guillochon's PR, etc.

**Comment by keflavich** [comment]
@bsipocz didn't work for me:
```
$ issue2pr keflavich:simbad_issue891 891 --sourceuser keflavich --repo astroquery
Converting issue 891 of astropy/astroquery to pull request
Will request to pull branch keflavich:simbad_issue891 of user's astroquery repo to branch master of  astropy/astroquery
Enter your GitHub password so that API requests aren't as severely rate-limited...
Password:
```
Oh well.

**Comment by bsipocz** [comment]
@keflavich - Hmm, that's bad. I'll have a look at it (I've used different calling args), and the worst case open an issue in the tools repo.

**Comment by keflavich** [comment]
Closed by #892
